study.md

```diff
@@ -50,7 +50,29 @@ A kitchen has many items.
     rack, and cinnamon, nutmeg, and vanilla on another shelf.
 
 ```json
-<!-- your answer here -->
+"kitchen": {
+  "cupboard":     {
+    "bowls",
+    "plates"
```

Comment:
careful here, if cupboard's value is another object, 'bowls' and 'plates' would be keys that pointed to values. easier to make cupboard an array that has 'bowls' and 'plates' in it.